examples/chart/teleport/values.yaml

```diff
@@ -22,18 +22,16 @@ image:
 
 labels: {}
 
-
 # Teleport Proxy configuration
 proxy:
   tls:
-    # We assume TLS is terminated in front of the proxy by default
+    # We assume that Teleport will handle TLS termination by default
     enabled: true
     # Set this to false if you want to use Teleport's generated self-signed certificates
     usetlssecret: true
```

Comment:
When using Teleport self-signed certs do customers still get a browser warning?  

Now we have this in our docs, it would be good to get a demo / video of this setup.

Reply:
> When using Teleport self-signed certs do customers still get a browser warning?

Yes. There isn't really a good way to avoid this when just using Teleport. The simplest way to do it in Kubernetes is to disable TLS termination via Teleport itself, then let an `Ingress` with [`cert-manager`](https://github.com/jetstack/cert-manager) correctly set up automatically provision your certs for you. How to set that up is probably out of scope for Teleport docs, though.

Reply:
I know it's out of scope for this PR, but I wonder what the best practices are for this. Chrome is really locking down on self-signed certs, and would be good to have a nice story / instructions for this. @awly Do you know of any other OSS project that has his same issue?

Reply:
Something that works for _everything_ is pretty hard. Maybe we could provide an easier way to get the self-signed web CA out of Teleport and give instructions on how to import that into the local browser store. It's all just pain though.

It's either that, or we make it easier to run without TLS at all - not great for a security-focused product.

Reply:
We can also consider compiling a [LetsEncrypt client](https://godoc.org/golang.org/x/crypto/acme/autocert) into teleport.
For example, a proxy can attempt to get certs for itself before falling back to self-signed certs.

Examples of this:
- https://caddyserver.com/docs/automatic-https
- https://github.com/haproxytech/haproxy-lua-acme
- https://perkeep.org/doc/server-config
- https://docs.gitea.io/en-us/https-setup/#using-lets-encrypt
- etc https://pkg.go.dev/golang.org/x/crypto/acme/autocert?tab=importedby

Reply:
+1 I like this, but to not block the fix. Let's put it into an issue and work on another PR.

Reply:
https://github.com/gravitational/teleport/issues/4117

Reply:
The other alternative for local testing (that @benarent just reminded me of) is something like https://github.com/FiloSottile/mkcert